wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -12,8 +12,10 @@
 
         <div class="row row-flush">
             <div class="left col9 header-title">
-                <h1 class="icon icon-doc-empty-inverse">
-                {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span></h1>
+                <h1>
+                  {% icon "doc-empty-inverse" %}
```

Comment:
missing `class_name="header-title-icon"`

other than that, works like a charm!